content/code-security/codeql-cli/codeql-cli-manual/database-analyze.md

```diff
@@ -256,7 +256,7 @@ Download any missing queries before analyzing.
 
 A list of threat models to enable or disable.
 
-The argument is the name of a threat model, optionally preceeded by a
+The argument is the name of a threat model, optionally preceded by a
```

Comment:
I've followed up about this: I can't guarantee a timeframe for a fix but thanks for pointing it out!

Reply:
I've fixed this internally and it'll go out in the next couple of weeks: unfortunately because of the way this is generated I wasn't able to add you as co-committer, but thank you for pointing it out!